cellbase-client/src/test/java/org/opencb/cellbase/client/rest/VariantClientTest.java

```diff
@@ -139,6 +145,77 @@ public void getAllConsequenceTypes(CellBaseClient cellBaseClient) throws Excepti
         assertNotNull(response.firstResult(), "List of all the consequence types present should be returned");
     }
 
+    @Test
+    public void testSearchSnpBydbSnpId() throws Exception {
+        Assumptions.assumeTrue(GitRepositoryState.get().getBranch().equals("TASK-5789"));
```

Comment:
So, this test won't work as soon as you merge the PR? Kind of weird.
And this is not easy, as the "v5.8.3-SNAPSHOT" is the only one containing the data you're trying to query and test.
What about, configuring this to "v5.8" , which is currently "v5.8.2" and adding an "AssumeTrue" that checks that the actual version (from meta/about) is at least 5.8.3?

We have similar issues in opencga:
https://github.com/opencb/opencga/blob/develop/opencga-storage/opencga-storage-core/src/test/java/org/opencb/opencga/storage/core/utils/CellBaseUtilsTest.java#L114-L122

Reply:
I'll work on improving that; the plan is to update these JUnit tests in the next release, since they're currently relying on an ad-hoc CellBase server.